unit-tests/unit-tests-rsutil.cpp

```diff
@@ -0,0 +1,31 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
This (`unit-tests/unit-tests-rsutil.cpp`) is a repetition of the C code.

I honestly think a better solution would have been a Python script that calls these functions. No CMakes, no compilation, nothing weird, and it could have been included in LibCI.